### PR TITLE
Remove scrollbar in table

### LINKED ISSUE #2873: scrolling using keyboard shortcuts is
When you hit 'j' or 'k' to scroll down/up on a long worksheet, the worksheet scrolls with each key press, which leads to jerky behavior.  The desired behavior (as it was before) is to scroll only when the selected bundle cursor is near the edge and scroll a more significant amount so that scrolling happens less frequently.

### LINKED ISSUE #2875: tables shouldn't have scrollbars
There should only be one scrollbar for the worksheet, but now for some reason, tables have scroll bars too, which makes it very confusing.
Screenshot from 2020-09-22 01-17-28

### PR DESCRIPTION
**Pull request by yuqijin**
### Reasons for making this change

Remove the scrollbar in the table. The table cannot be scrolled anymore.

### Related issues

fixes #2875 ; reopens #2082

### Screenshots
No scroll bar: 
Screen Shot 2020-10-03 at 3 01 55 PM

### Checklist

* [x] I've added a screenshot of the changes, if this is a frontend change
* [x] I've added and/or updated tests, if this is a backend change
* [x] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
* [x] I've updated [docs](https://github.com/codalab/codalab-worksheets/tree/master/docs), if needed

### CODE REVIEW

File: frontend/src/components/worksheets/items/TableItem/TableItem.js [hunk 1]
```diff
@@ -328,6 +333,11 @@ class _TableContainer extends React.Component {
 const styles = (theme) => ({
     tableContainer: {
         position: 'relative',
+        '&::-webkit-scrollbar': {
```
Comment (epicfaace) [inline frontend/src/components/worksheets/items/TableItem/TableItem.js]: Why do this instead of `overflow: hidden`?

Comment (yuqijin) [reply frontend/src/components/worksheets/items/TableItem/TableItem.js]: I just switched the method! Thanks for your advice.

Comment (epicfaace) [reply frontend/src/components/worksheets/items/TableItem/TableItem.js]: Is there a chance we might cut off text in the table, this way? Or is that not going to be an issue?

Comment (yuqijin) [reply frontend/src/components/worksheets/items/TableItem/TableItem.js]: I think you can scroll left and right to see the text if it is overflow. In the gif you can see no text is cut off.

Comment (yuqijin) [reply frontend/src/components/worksheets/items/TableItem/TableItem.js]: Screen Shot 2020-09-30 at 2 07 52 PM

Comment (yuqijin) [reply frontend/src/components/worksheets/items/TableItem/TableItem.js]: It is proved that the `overflow` method will cut off the text. So I switched back to the specific webkit-scrollbar method.

**Review by epicfaace — APPROVED** [review]

**Pull request merged**